CTestConfig.cmake

```diff
@@ -13,7 +13,8 @@
 #
 set(CTEST_PROJECT_NAME "pika")
 set(CTEST_NIGHTLY_START_TIME "00:00:00 GMT")
+set(CTEST_DROP_METHOD "https")
 
-set(CTEST_SUBMIT_URL
-    "https://cdash.cscs.ch/submit.php?project=${CTEST_PROJECT_NAME}"
-)
+set(CTEST_DROP_SITE "cdash.cscs.ch")
+set(CTEST_DROP_LOCATION "/submit.php?project=pika")
```

Comment:
should we still use `${CTEST_PROJECT_NAME}` here?

Reply:
as long as `${CTEST_PROJECT_NAME}` expands to pika, then it doesn't really matter as far as the dashboard is concerned